### PR TITLE
Exception handling

### PR DESCRIPTION
**Pull request by Justin-JKim**

### CODE REVIEW

File: src/main/resources/templates/error-403.html [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Error 403</title>
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css">
+    <style>
```
Comment (RobertDeWolf7) [inline src/main/resources/templates/error-403.html]: Can you do this in a stylesheet

File: src/main/resources/templates/error-404.html [hunk 2]
```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Error 404</title>
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css">
+    <style>
```
Comment (RobertDeWolf7) [inline src/main/resources/templates/error-404.html]: Same thing here

File: src/main/resources/templates/error.html [hunk 3]
```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Error</title>
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css">
+    <style>
```
Comment (RobertDeWolf7) [inline src/main/resources/templates/error.html]: One stylesheet for all of these sounds like a good idea

File: src/main/resources/templates/error-403.html (hunk 1 above)
Comment (Justin-JKim) [reply src/main/resources/templates/error-403.html]: I added a css file that all error pages reference.

File: src/main/resources/templates/error-404.html (hunk 2 above)
Comment (Justin-JKim) [reply src/main/resources/templates/error-404.html]: I added a css file that all error pages reference.

File: src/main/resources/templates/error.html (hunk 3 above)
Comment (Justin-JKim) [reply src/main/resources/templates/error.html]: I added a css file that all error pages reference.

**Review by RobertDeWolf7 — APPROVED** [review]

**Pull request merged**